.vsts-ci.yml

```diff
@@ -37,10 +37,10 @@ stages:
         pool:
           ${{ if ne(variables['System.TeamProject'], 'internal') }}:
             name: NetCore-Public
-            demands: ImageOverride -equals Build.Windows.10.Amd64.VS2019.Open
+            demands: ImageOverride -equals windows.vs2019.amd64.open
```

Comment:
Is there a link or reference we can drop in here for an FYI?

Reply:
[Added a link](https://github.com/dotnet/sign/pull/440/commits/45a680bd9dc8f933a9b4ee1f09f70be3dcf308fd) to https://helix.dot.net/.